users/utils.py

```diff
@@ -0,0 +1,25 @@
+import json,jwt,requests
+
+from django.http  import JsonResponse 
+
+from my_settings  import ALGORITHM, SECRET_KEY
+from .models import User
+
+def login_decorator(func):
+    def wrapper(self, request):
+        try:
+            acess_token    = request.headers.get('Authorization', None)
+            amount         = jwt.decode(acess_token,SECRET_KEY,ALGORITHM)
+            users          = User.objects.get(email=amount['id'])
```

Comment:
`email`에 `amount["id"]`로 조회하고 있는데, 이상해보이네요.
제대로 동작하는 건가요?

```suggestion
            users          = User.objects.get(email=amount['id'])
```